dingo/bindings/bindings.cpp

```diff
@@ -103,6 +103,11 @@ double HPolytopeCPP::apply_sampling(int walk_len,
    HP.set_InnerBall(CheBall);
    starting_point = inner_point2;
    std::list<Point> rand_points;
+   NT variance = 1.0;
+   NT a = NT(1)/(NT(2)*variance);
+   int dim = 50;
+   Point c(dim);
+   c = GetDirection<Point>::apply(dim, rng, false);
```

Comment:
this part is probably a leftover, right?

Reply:
These are used as parameters to call the walks for sampling, like rand_points which was already defined just above and used in the walks. I guess I could define them just in the part where they are used, but maybe they are reused when other walks are exposed in the future.

Reply:
why `dim` is fixed to `50`?

Reply:
I guess that may not be needed, I was following the format of the log concave examples where dim is set to 50. I can delete that part.

Reply:
you should rather use `d` here which is the dimension of the polytope, otherwise you will get wrong results (maybe also a compilation error). Also these constants should be defined inside the relevant if..then..else blocks, i.e. `a` in gaussian hmc and `c` in exponential hmc.

Reply:
Sure, will apply those modifications and submit another PR.

Reply:
Please, do not submit another PR! Just create a new commit in the branch associated with this PR. The modifications will be visible here and we can continue reviewing. Please also have a look at the comment regarding tests.